signin/index.html

```diff
@@ -0,0 +1,54 @@
+<!doctype html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>signin</title>
+    <link rel="stylesheet" href="signin.css" />
+  </head>
+  <body>
+    <header>
+      <div class="sign">
+        <a href="../index.html"><img src="../image/logo2.png" /></a>
+        <p class="signup2">
+          회원이 아니신가요?
+          <a href="../signup/index.html"
+            ><span class="signup">회원가입하기</span></a
+          >
+        </p>
+      </div>
+      <form>
+        <div class="email">
+          <p>이메일</p>
+          <div class="box2">
+            <label for="email"></label>
```

Comment:
label이 쓸모가 있도록 해당 텍스트를 label 내에 넣어주세요.
```suggestion
          <div class="box2">
            <label for="email">이메일</label>
```

Reply:
확인 후 수정하였습니다 !